client/client_test.go

```diff
@@ -86,7 +86,7 @@ func TestNewClientWithOpsFromEnv(t *testing.T) {
 		},
 	}
 
-	defer env.PatchAll(t, nil)()
+	env.PatchAll(t, nil)()
```

Comment:
DOH! I know what's wrong (probably!); this should of course be

```suggestion
	env.PatchAll(t, nil)
```

Otherwise, we're calling the `cleanup` _immediately_ 🤦